lib/statsd.js

```diff
@@ -359,7 +359,13 @@ Client.prototype.sendMessage = function (message, callback) {
   }
 
   if (!this.socket) {
-    return callback(new Error('Socket not created properly. Check previous errors for details.'));
+    const error = 'Socket not created properly. Check previous errors for details.';
+    if (callback) {
+      return callback(new Error(error));
+    } else {
+        console.error(error);
+        return;
```

Comment:
Can you remove this "return", as the style in here hasn't been to include in this case, and can you unindent the console.error line by 2?

Thanks!  Once that is done I can merge this in, very clear explanation you gave in the bug on what is wrong here.

Reply:
Hey @bdeitte 

I fixed the indent and removed the return in bd2b68e5ae5f2c84f2fa2748371b58136dafcb46.

However, without the return, this is going to error out on the [socket send](https://github.com/brightcove/hot-shots/pull/153/files#diff-cfdf1e0afcbe7fb32697031bec3cc7deR391). Is that what we want?

Reply:
Oh you're right there, the inline function got me there!  I'll add back in as I merge, thanks.